mne/viz/backends/_utils.py

```diff
@@ -238,8 +244,7 @@ def _qt_raise_window(widget):
         widget.raise_()
 
 
-def _qt_is_dark(widget):
+def _qt_is_dark(win):
```

Comment:
Why change this? I think it might break `mne-qt-browser`. QMainWindow inherts from QWidget, which has the `.window()` method, so it seems harmless to keep the `.window()` below.

Reply:
> Why change this?

It's simpler and it removes the indirection? 

I was not aware of the dependency in `mne-qt-browser`, I'll revert.